model/distro/distro.go

```diff
@@ -346,6 +347,13 @@ func (s *PlannerSettings) GetGenerateTaskFactor() int64 {
 	return s.GenerateTaskFactor
 }
 
+func (s *PlannerSettings) GetNumDependentsFactor() int64 {
+	if s.NumDependentsFactor <= 0 {
+		return 10
```

Comment:
Preference for this to still default to 1, even if that's different than what we have now. we could run an update to set this to 10 in the DB so that it's ready when we deploy.